### ISSUE TITLE
Can i run more than one nodeid?

### ISSUE THREAD

**Issue by mizaty** [recitativonika/blockless-bless-network-bot#1]
Can i run more than one node id at the same time? Coz when i put second nodeid in the new line it doean't react

**Comment by recitativonika** [comment]
no, I only test using one node id. maybe I will make it to support multi nodeid support later

**Comment by recitativonika** [comment]
Now support multi nodeid.

**Comment by mizaty** [comment]
Bro you're legend,do you have any telegram contact or whatever i can catch up

**Comment by recitativonika** [comment]
Please pull the repo again,  I forgot to add ipfetch.

For my contact,  sorry I can give to anyone for my privacy sake.

**Comment by mizaty** [comment]
It works now thanksyou, its oke mate i respect that
